civictechprojects/static/css/partials/_MainHeader.scss

```diff
@@ -21,11 +25,43 @@
 }
 
 .MainHeader-links {
+  @media screen and (max-width: $break-small) {
+    display: none;
+  }  
   float: right;
 }
 
+.MainHeader-hamburger {
+  @media screen and (max-width: $break-small) {
+    display: block;
+  }  
+  display: none; 
+}
+
+.MainHeader-navDrawer {
+  margin-top: 25px;
+  width: 43.72vw;
+  min-width: 200px;
+}
+
+.MainHeader-drawerSpan {
+  margin-top: 10px;
```

Comment:
inline elements like span don't take top/bottom margins, only left/right, and it does feel like the hamburger menu items are pretty tightly grouped on a phone as a result. I'd like to see the clicktargets be larger and the items be spaced a little further apart vertically to make tapping with a finger easier. 

Changing these to block elements (a div, most likely) and using padding instead of margin should resolve the spacing and the clicktarget size.

Reply:
Thanks for reviewing my commits. I'll make sure to make these changes within the next few days